openstates/ok/people.py

```diff
@@ -38,23 +38,21 @@ def _extract_phone(self, office_info):
 
         return phone
 
-    def _extract_email(self, doc):
-        xpath = '//div[@class="districtheadleft"]' \
-                + '/b[contains(text(), "Email:")]' \
-                + '/../following-sibling::div' \
-                + '/script/text()'
-        script = doc.xpath(xpath)
-        if not script:
-            return ''
-        script = script[0]
-        line = filter(
-            lambda line: '+ "@" +' in line,
-            script.split('\r\n'))[0]
-        parts = re.findall(r'"(.+?)"', line)
-
-        email = ''.join(parts)
-
-        return email if validate_email_address(email) else ''
+    def _get_rep_email(self, district):
+        # Get the email address from an input on the contact page
+        url = "https://www.okhouse.gov/Members/Contact.aspx?District=" + district
+        page = self.curl_lxmlize(url)
+
+        try:
+            email_node = page.get_element_by_id('txtMemberEmail')
+            email = email_node.value
+        except KeyError:
+            email = ''
```

Comment:
Can we use `None` instead of `''` here? Doesn't matter much, but it'll be clearer to others later that we weren't, say, planning to append anything to the empty string variable later.